Client/lib/testAPI.html

```diff
@@ -0,0 +1,52 @@
+<!-- 
+
+
+things to look into 
+loadGeoJson 
+
+
+-->
+
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <style type='text/css'>
+      html, body, #gMap { 
+        height: 100%; 
+        margin: 0; 
+        padding: 0;
+        cursor: url("https://maps.gstatic.com/mapfiles/openhand_8_8.cur");
+      }
+      .gMapInactiveButton {
+        background: rgba(70,70,70,0.5);
+      }
+      .gMapInactiveButton:hover {
+        background: rgba(33,33,33,0.6);
+      }
+      .gMapActiveButton {
+        background: rgba(33,33,33,0.9);
+      }
+
+    </style>
+    
+    <script type='text/javascript'
+      src='https://maps.googleapis.com/maps/api/js'>
+    </script>
+    <script type='text/javascript'
+      src='gMap.js'>
+    </script>
+    <script type='text/javascript'>
```

Comment:
I wanted to keep the CSS separate so the other team members can edit the style without having to worry about the gMap.js code.
As for the script tags unless we use require or something alike I would imagine we will keep it in the main html file.